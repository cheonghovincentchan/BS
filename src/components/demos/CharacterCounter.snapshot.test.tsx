/**
 * snapshots unit testing docs https://jestjs.io/docs/tutorial-react#snapshot-testing
 */
import renderer from 'react-test-renderer';
import { RecoilRoot } from 'recoil';
import CharacterCounter from './CharacterCounter';

describe('CharacterCounter snapshot unit test', () => {
  const component = renderer.create(
    <RecoilRoot>
      <CharacterCounter />
    </RecoilRoot>,
  );
  it('bacic test', () => {
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
