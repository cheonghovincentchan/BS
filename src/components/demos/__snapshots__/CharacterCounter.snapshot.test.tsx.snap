// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterCounter snapshot unit test bacic test 1`] = `
<div>
  <div>
    <input
      data-testid="text-input"
      onChange={[Function]}
      type="text"
      value=""
    />
    <br />
    Echo: 
  </div>
  <span
    data-testid="text-show"
  >
    Character Count: 
    0
  </span>
</div>
`;
